### PR TITLE
Eduardosancho feature add webpage

### PR DESCRIPTION
**Pull request by eduardosancho**
# What was done in this PR:

```
-Add project name and description on ReadMe
-Add index.html and a heading
-Add styles.css and give the heading a golden background
-Add test.md to .gitignore
-Include all the syntax suggestions from linters to index.html and styles.css
```

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <!--<meta http-equiv="X-UA-Compatible" content="IE=edge">-->
```
Comment (mmsesay) [inline index.html:5]: - [ ] [OPTIONAL] Please remove commented codes to keep your entire code clean.

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @eduardosancho , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mmsesay** [comment]
- [ ] Good job in setting up the linter configuration for your project :clap: 
However, there are some linter errors. I believe we can try to fix them.

**Review by eduardosancho — COMMENTED** [review]
The problem was caused by the hint-report/ folder, so I added it to .gitignore,
and surprisingly it passed the webhint test, which was the only one missing.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+node_modules/
+test.md
+hint-report/
```
Comment (Gambit142) [inline .gitignore:3]: - [ ] (Optional) I would like to advise you to delete the `.hint-report` file and use the correct linters for any other _HTML_ and _CSS_ project. You can read the instructions in this [article](https://github.com/microverseinc/linters-config/tree/master/html-css) to properly set up linters in your project

**Review by Gambit142 — APPROVED** [review]
Hi @eduardosancho 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**